src/visualization/recipes_makie.jl

```diff
@@ -4,10 +4,12 @@
 # See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
 @muladd begin
 
+default_colormap() = :inferno
```

Comment:
I think it makes sense to define such a default, but why restrict it to Makie? I suggest to move it to neutral territory (maybe `visualization/visualization.jl`) and add a comment, e.g.,
```julia
# Default color map used in plotting recipes for Makie.jl and Plots.jl
```
Then, also replace all hardcoded instances of `:inferno` in `recipes_plots.jl`. What's your take on this?

Reply:
I don't see any instances of `:inferno` in `recipes_plots.jl`. If we really want one unified default behavior, we should go through and add the colormap to all the Plots routines. 

I'd prefer to change `default_colormap()` to `default_Makie_colormap` for this PR and create an issue for colormap unification if that's OK.

Reply:
Ah, right, `:inferno` is already the default for Plots.jl itself. No, never mind then, no need for extra code here I think. Just leave it as it is.